infra/docker-swarm/ansible/roles/testground-services/tasks/main.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Create Testground overlay control network
+  docker_network:
+    name: control
+    driver: overlay
+    attachable: yes
+    enable_ipv6: no
+    ipam_config:
+      - subnet: 10.1.0.0/16
+
+- name: Pull ipfs/testground image
+  docker_image:
+    name: "ipfs/testground:latest"
+    source: pull
+    force_source: yes
+    state: present
+
+- name: Pull Redis image
```

Comment:
I think this pull is redundant. This will pull the image in the master, not in the machine where Redis is scheduled.

Reply:
You're right - I was confused because this is generally needed when you do `docker_container`, but here we actually schedule a `docker_swarm_service`, so it should do the pull.